estuary_updater/handlers/freshmaker.py

```diff
@@ -89,16 +89,20 @@ def build_state_handler(self, msg):
         :param dict msg: a message to be processed
         """
         build = msg['body']['msg']
-        self.create_or_update_build(build)
+        self.create_or_update_build(build, msg)
 
-    def create_or_update_build(self, build):
+    def create_or_update_build(self, build, msg):
         """
         Use the Koji Task Result to create or update a ContainerKojiBuild.
 
         :param dict build: the build being created or updated
```

Comment:
Could you please update the docstring if you decided to keep accepting a new parameter based on my comment above?